### PR TITLE
Port MemorySelectRow to PF4 components

### PR DESCRIPTION
**Pull request by KKoukiou**
* [x] Blocked on https://github.com/patternfly/patternfly-react/pull/5710
* [x]  Blocked on https://github.com/patternfly/patternfly-react/pull/5740
* [x]  Blocked on https://github.com/patternfly/patternfly-react/pull/5878

Other Related issue: https://github.com/patternfly/patternfly-react/issues/5968

### CODE REVIEW

**Comment by garrett** [comment]
The sliders work as expected _at first_. But then, when you adjust the bottom slider, the top acts really odd when you try to resize it.

Here's a video of what I'm talking about:
https://user-images.githubusercontent.com/10246/123671878-08f8e280-d83f-11eb-9b59-e69b3cadf5cb.mp4

**Comment by KKoukiou** [comment]
@garrett thanks for finding this issue - I reported it upstream on https://github.com/patternfly/patternfly-react/issues/5968

Luckily I was able to find a workaround.

File: src/components/vm/overview/memorySelectRow.jsx [hunk 1]
```diff
@@ -37,59 +39,58 @@ class MemorySelectRow extends React.Component {
     }
 
     static getDerivedStateFromProps(nextProps, prevState) {
-        if (nextProps.value !== prevState.memory && !prevState.inputHasFocus)
+        if (nextProps.value != prevState.memory && !prevState.inputHasFocus)
```
Comment (martinpitt) [inline src/components/vm/overview/memorySelectRow.jsx:42]: Out of interest, was that deliberate? Could this be a string vs. number comparison or so? Or accidental?

**Review by martinpitt — DISMISSED** [review]
One more down, thanks! So now it's just the bunch in bootOrder.jsx, everything else looks like pure PF4 now?

File: src/components/vm/overview/memorySelectRow.jsx (hunk 1 above)
Comment (KKoukiou) [reply src/components/vm/overview/memorySelectRow.jsx:42]: Probably an accident, but I wrote this code a few months ago so I would like to recheck later today :) If it's accidental I will repost.

Comment (martinpitt) [reply src/components/vm/overview/memorySelectRow.jsx:42]: Not a big deal either way, I was just curious. (Approval stands)

Comment (KKoukiou) [reply src/components/vm/overview/memorySelectRow.jsx]: Reposted with reverting this small change.

**Review by martinpitt — APPROVED** [review]

**Comment by KKoukiou** [comment]
fedora-33/firefox is no longer in the testmap.

**Pull request merged**